lib/modules/manager/gomod/artifacts.ts

```diff
@@ -31,7 +31,7 @@ import { getExtraDepsNotice } from './artifacts-extra';
 const { major, valid } = semver;
 
 function getUpdateImportPathCmds(
-  updatedDeps: PackageDependency[],
+  updatedDeps: Upgrade[],
   { constraints }: UpdateArtifactsConfig,
 ): string[] {
   // Check if we fail to parse any major versions and log that they're skipped
```

Comment:
For some reason, when we're seeing a replacement, we get the wrong data here :thinking: